section1/ex7.rb

```diff
@@ -0,0 +1,8 @@
+print "How old are you?"
```

Comment:
Challenge yourself: When you run this and test, your user's input is right next to the print out. Would you keep it this way or update?